src/domain/collaboration/callout/callout.service.spec.ts

```
@@ -0,0 +1,238 @@
+      await service.getActivityCountBatch([callout]);
+
+      // The same object should now have activity set
+      expect(callout.activity).toBe(7);
+    });
+
+    it('should parallelize comment RPC calls', async () => {
+      const comment1 = makeCommentCallout('c-1');
+      const comment2 = makeCommentCallout('c-2');
+
+      // Track call order to verify parallelism
+      let callCount = 0;
```

Biome (2.3.14)

[error] This variable callCount is unused.

Unused variables are often the result of typos, incomplete refactors, or other sources of bugs.
Unsafe fix: If this is intentional, prepend callCount with an underscore.


(lint/correctness/noUnusedVariables)